doc/howtos/image-installer/init-tpm

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -ex
```

Comment:
Might it be a good idea to have these scripts in `intel-iot-refkit/scripts/image-installer` instead of `intel-iot-refkit/doc/howtos/image-installer`?